rero_ils/utils.py

```diff
@@ -80,3 +81,63 @@ def get_agg_config(index_name, field):
             current_app.config.get('RERO_ILS_DEFAULT_AGGREGATION_SIZE')
         )
     ))
+
+
+def git_get_current_branch():
```

Comment:
I'm not sure that is robust. I will probably do call a `git` command directly or use a python package to deal with git. Something like:
```python
import subprocess
label = subprocess.check_output(["git", "describe"]).strip()
```